canonical-webteam/global-nav
# Global nav v1.0.0 merge proposal

## Done

- Just merging all the changes from the `new-global-nav` branch, which have been Code and QA +1'd individually

## QA

- Pull code
- Run `yarn build`
- Open `index.html`
- Check that the global nav example works as expected
